src/components/GenreSelector/GenreSelector.jsx

```diff
@@ -16,7 +16,7 @@ const GenreSelector = ({ genres, selectedGenre = "All", onSelect }) => {
           style={{
             backgroundColor: "#232323",
             color: "#FFFFFF",
-            "border-bottom":
+            borderBottom:
               "3px solid " + (genre === currentGenre ? "#F65261" : "#424242"),
```

Comment:
Don't use condition inline styling, bad for perfomance. User className toggle

Reply:
fixed in react-mentoring-lis-m05 branch